### PR TITLE
chore: setup build process

### PR DESCRIPTION
**Pull request by fforootd**
This should work as initial pipeline.

A refactoring will be necessary later.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
+name: Release
+on: push
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  REGISTRY: docker.pkg.github.com
+  IMAGE: zitadel
+  NODE_VERSION: '12'
+  GO_VERSION: '^1.13.1'
```
Comment (livio-a) [inline .github/workflows/release.yml:9]:
```suggestion
  GO_VERSION: '^1.14.1'
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        name: angular
+        path: hodor.txt
+  
+  go: # TODO Implement proper build and cache
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2-beta
+      with:
+        go-version: '^1.13.1'
```
Comment (livio-a) [inline .github/workflows/release.yml:36]:
```suggestion
        go-version: ${{ env.GO_VERSION }}
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    - run: go test -race ./...
+    - run: go build -o zitadel main.go
```
Comment (livio-a) [inline .github/workflows/release.yml:38]:
```suggestion
    - run: go build -o zitadel cmd/zitadel/main.go
```

File: main.go [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package main
```
Comment (livio-a) [inline main.go:1]: file is not needed as we already have a working main.go

File: .github/workflows/release.yml (hunk 3 above)
Comment (livio-a) [reply .github/workflows/release.yml:38]: might have to change further steps to point to this path

File: main.go (hunk 4 above)
Comment (fforootd) [reply main.go]: ok i delete this

**Pull request merged**